src/app/constructor/dashboard/dashboard-constructor.component.ts

```diff
@@ -10,38 +10,46 @@ import { Device } from '../../shared/devices/device.model';
 export class DashboardConstructorComponent {
     @Input() canBeActive: boolean;
     @Input() default: string;
+    @Input() dashboardSubView: DashboardViewInfoDto;
```

Comment:
Word "dashboard" in "dashboardSubView" variable name is redundant, since you are already in dashboard component. Also name ("sub" prefix) means that component cannot exist without the parent view, which is not true. I would recommend you to use just generic "data" name for this variable.
@Input() data: DashboardViewInfoDto ?